utils/ProvinceHelper.js

```diff
@@ -0,0 +1,45 @@
+const PROVINCE = [
```

Comment:
You can just take the `provincesWithCities` array from 
https://github.com/agallio/ina-covid-bed/blob/afeb984bf2d45cd7b5af0a9485d33b8efb8cd144/pages/api/bed.js#L5

And then map it to new array that contains every provinces `value`.